### PR TITLE
Elixir LS with Vim

### PR DESCRIPTION
**Pull request by mhanberg**

### CODE REVIEW

File: _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: How to use Elixir LS with Vim
+date: 2018-10-07 09:00:00 -04:00
```
Comment (mhanberg) [inline _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md]: Remember to update this to the correct date when you publish.

File: _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+```vim
+" .vimrc
+
+let g:ale_elixir_elixir_ls_release = '<your path to elixir-ls>/rel'
+```
+
+### Now what?
+
+I would familiarize yourself with the [features of ALE](https://github.com/w0rp/ale#usage) and decide how you want to use incorporate them into your workflow. ALE doesn't prescribe any keymappings, so feel free to experiment to see what works best for you!
```
Comment (mhanberg) [inline _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md]: ~use~ incorporate

File: _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+permalink: /:categories/:year/:month/:day/:title/
+---
+
+### What is Elixir LS?
+
+[Elixir LS](https://github.com/JakeBecker/elixir-ls) by Jake Becker is the language server implementation for Elixir.
+
+### What is a language server?
+
+If you've been following the story of Visual Studio Code, there is a chance you've heard of another recent creation from Microsoft: the Language Server Protocol.
```
Comment (rpherbig) [inline _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md]: Maybe add a link to https://langserver.org/ or https://github.com/Microsoft/language-server-protocol ?

File: _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md (hunk 2 above)
Comment (rpherbig) [inline _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md]: Does ALE (or Mitch) have some 'default' or 'suggested' keymappings?

File: _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+[Elixir LS](https://github.com/JakeBecker/elixir-ls) by Jake Becker is the language server implementation for Elixir.
+
+### What is a language server?
+
+If you've been following the story of Visual Studio Code, there is a chance you've heard of another recent creation from Microsoft: the Language Server Protocol. 
+
+>The Language Server Protocol (LSP) defines the protocol used between an editor or IDE and a language server that provides language features like auto complete, go to definition, find all references etc.
+
+This allows creators to build universal "language servers" that can be used by any text editor. These servers can provide features like code completion, go to definition, and code diagnostics.
```
Comment (rpherbig) [inline _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md]: `can provide features like...` is this a restatement of the quote above?

File: _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md (hunk 3 above)
Comment (mhanberg) [reply _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md]: Good idea, thought i did that.

File: _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md (hunk 4 above)
Comment (mhanberg) [reply _posts/2018-10-07-how-to-use-elixir-ls-with-vim.md]: I was reading this over last night and I also made that realization. I will probably cut my restatement.

**Pull request merged**